README.md

```diff
@@ -138,37 +154,54 @@ Logging:
   File: pkgr-install.log
 ```
 
-TODO:
+## Pkgr and [Packrat](https://rstudio.github.io/packrat/)
```

Comment:
Hey @dpastoor , I need you to look at this section pretty closely. I'm not sure I fully understand all of the nuance here.